users/templates/user_notifications_list.html

```diff
@@ -1,36 +1,46 @@
 {% extends 'base.html' %}
 
 {% load i18n %}
+{% load static %}
 {% load data_structure_element_selectors %}
 
+{% block extra_head %}
+  <link rel="stylesheet" href="//code.jquery.com/ui/1.12.1/themes/smoothness/jquery-ui.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="stylesheet" type="text/css" href="{% static 'users/style.css' %}">
```

Comment:
All these stylesheets does not have SRI.

Reply:
done